### PR TITLE
Mail update 160117

### PR DESCRIPTION
**Pull request by piotrdrag**

### CODE REVIEW

File: mail/chrome/messenger/preferences/advanced.dtd [hunk 1]
```diff
@@ -89,7 +89,7 @@
...
 <!ENTITY enableOCSP.accesskey "e">
 
-<!ENTITY viewCertificates.label "Wyświetl certyfikaty">
-<!ENTITY viewCertificates.accesskey "W">
-<!ENTITY viewSecurityDevices.label "Urządzenia zabezpieczające">
-<!ENTITY viewSecurityDevices.accesskey "U">
++<!ENTITY manageCertificates.label "Zarządzaj certyfikatami">
++<!ENTITY manageCertificates.accesskey "Z">
++<!ENTITY viewSecurityDevices.label "Urządzenia zabezpieczające">
++<!ENTITY viewSecurityDevices.accesskey "U">
```
Comment (splewako) [inline mail/chrome/messenger/preferences/advanced.dtd]: This would cause XML parsing errors and needs to be corrected.

**Comment by splewako** [comment]
Please fix second commit message also.

File: mail/chrome/messenger/preferences/advanced.dtd (hunk 1 above)
Comment (piotrdrag) [inline mail/chrome/messenger/preferences/advanced.dtd]: Fixed.

**Comment by piotrdrag** [comment]
Both fixed.

**Comment by splewako** [comment]
Thank you for the patches, landed as https://hg.mozilla.org/l10n-central/pl/rev/e4df1557c518 and https://github.com/aviarypl/mozilla-l10n-central-pl/compare/783c20e30952...26d97448be16

**Pull request closed without merging**